templates/base/generators/stores/contextExample.js.hbs

```diff
@@ -0,0 +1,44 @@
+import React, { useContext } from "react";
+import { getContext } from "smart-context";
+
+const ContextExample = () => {
```

Comment:
This will go in Documentation and the future sample app we will create. This is not needed in generator.